src/plugins/intel_npu/tools/compile_tool/CMakeLists.txt

```diff
@@ -12,6 +12,22 @@ if (NOT DEFINED PROJECT_NAME)
     return()
 endif()
 
+#
```

Comment:
maybe you just need to enable gflags build here
https://github.com/openvinotoolkit/openvino/blob/d3096afb11b9f2e87ff0165efe2630bd01f49a66/thirdparty/dependencies.cmake#L276-L280
if `ENABLE_INTEL_NPU_INTERNAL` is true ?